source/data-encryption.txt

```diff
@@ -12,59 +12,167 @@ Data Encryption
    :depth: 2
    :class: onecol
 
-|service| offers encryption features to protect data while in transit, at rest,
-and in use to safeguard data through its full lifecycle.
+|service| offers several encryption features to protect data 
+while in transit, at rest, and in use to safeguard data through 
+its full lifecycle.
 
-{+service+} Features and Recommendations for Data Encryption
-------------------------------------------------------------
-
-Features 
-~~~~~~~~
+Features
```

Comment:
I like what you did here simplifying/reducing the heading levels, and I will replicate across other pages, but can we make this "Features for Data Encryption"? I ask because I know there's an effort from Sarah Lin to reduce duplicate headings in docs sets